### PR TITLE
Default labels

### LINKED ISSUE #500: Auto Labeling Configuration
The auto-label feature (the one where if an issue is submitted with no labels then the bot automatically adds `Price` and `Time` needs to be off by default, and configurable in the `ubiquibot-config.yml`

We also need full documentation around the config properties because I'm not even familiar with how to adjust this.

### Schema

- This is automatically added for every new issue opened without existing labels. 
- User specific config will be handled by a future bounty. 

```yml
default-labels:
    - "Time: <1 Hour"
    - "Priority: 0 (Normal)"
```

### PR DESCRIPTION
**Pull request by whilefoo**
Resolves #500 

QA issue: https://github.com/whilefoo/ubiquibot/issues/4

### CODE REVIEW

File: .github/ubiquibot-config.yml [hunk 1]
```diff
@@ -28,6 +28,15 @@ priority-labels:
...
     weight: 5
+default-labels:
+  global:
+    - "Time: <1 Hour"
+    - "Priority: 0 (Normal)"
+    - "Test"
+  users:
+    pavlovcik:
+      - "Time: <1 Hour"
+      - "Priority: 0 (Normal)"
```
Comment (pavlovcik) [inline .github/ubiquibot-config.yml:39]: Pretty cool you implemented this but I still fear that we'll need to remove this when we implement the database style config e.g. `/default-labels @pavlovcik "Time: <1 Hour" "Priority: 0 (Normal)"

File: README.md [hunk 2]
```diff
@@ -47,6 +47,40 @@ To test the bot, you can:
 2. Add a time label, ex: `Time: <1 Day`
 3. At this point the bot should add a price label.
 
+## Configuration
```
Comment (pavlovcik) [inline README.md:50]: You did a great job here thanks.

**Review by pavlovcik — COMMENTED** [review]
Code seems fine I just fear that we need to remove the user specific default label config code soon.

**Review by pavlovcik — APPROVED** [review]

File: .github/ubiquibot-config.yml [hunk 3]
```diff
@@ -28,6 +28,15 @@ priority-labels:
     weight: 4
   - name: "Priority: 4 (Emergency)"
     weight: 5
+default-labels:
+  global:
+    - "Time: <1 Hour"
+    - "Priority: 0 (Normal)"
+    - "Test"
+  users:
+    pavlovcik:
```
Comment (0xcodercrane) [inline .github/ubiquibot-config.yml:37]: oh no. seems better to use `ubiquity-bounties` instead of @pavlovcik  lol

**Review by 0xcodercrane — DISMISSED** [review]

File: .github/ubiquibot-config.yml (hunk 1 above)
Comment (pavlovcik) [inline .github/ubiquibot-config.yml:39]: There's one thing I forgot to highlight @whilefoo but https://github.com/ubiquity/ubiquibot/issues/500#issuecomment-1634429276 

Would you mind refactoring this config to be like this:

```suggestion
default-labels:
  - "Time: <1 Hour"
  - "Priority: 0 (Normal)"
  - "Test"
```

**Review by pavlovcik — CHANGES_REQUESTED** [review]

**Review by pavlovcik — APPROVED** [review]

**Review by Draeieg — APPROVED** [review]

**Comment by whilefoo** [comment]
what's stopping this from merging?

**Comment by pavlovcik** [comment]
> what's stopping this from merging?

That is unusual. GitHub iOS client UI doesn't tell me any useful information on it. @0xcodercrane do you know why

**Comment by 0xcodercrane** [comment]
> That is unusual. GitHub iOS client UI doesn't tell me any useful information on it. @0xcodercrane do you know why

Looks like related to unresolved conversations

**Pull request merged**